content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,129 +1,103 @@
-data, such as an ML model, by writing a simple `dvc.yaml` file that connects the
-pieces together:
-
-> `dvc.yaml` files can be written manually or generated with `dvc run`.
-
-```yaml
-stages:
-  train:
-    cmd: python train.py images/
-    deps:
-      - images
-    outs:
-      - model.pkl
-```
-
-> See [Data Pipelines](/doc/start/data-pipelines) for a comprehensive intro to
-> this feature.
-
-`dvc repro` can now execute the `train` stage for you. DVC will track all of its
-outputs (`outs`) automatically. Let's do that, and commit this project version:
+[Version control](https://en.wikipedia.org/wiki/Version_control) has become a
+staple in software engineering because it allows effective collaboration on
+source code. This means having a change history to traverse (commits),
+developing parallel features (branching and merging), peer-reviews (pull
+requests), release management, etc. Imagine using these features for data
+engineering and machine learning!
+
+Unfortunately, versioning tools like [Git](https://git-scm.com/) are designed to
+handle small text files. DVC begins by enabling the [tracking](#how-it-looks) of
+large datasets and other <abbr>data artifacts</abbr> by replacing them with
+small, human-readable [metafiles](/doc/user-guide/dvc-files-and-directories)
+that `git` can handle. The data itself is <abbr>cached</abbr> outside of Git.
+Other features of DVC build upon this foundation to tackle hurdles specific to
+data science.
+
+> 💡 Please see [Get Started](/doc/start) for a primer on all of DVC's features.
+
+Another limitation of source code management is storage. While some data and
+binary assets can exist in the Git repository, it's not usually recommended. And
+storage is severely limited by hosting services
+[like GitHub](https://docs.github.com/en/github/managing-large-files/what-is-my-disk-quota).
+DVC addresses this problem by integrating with
+[dedicated storage](#versioned-storage) platforms.
+
+## Summary of advantages
+
+- Reproducibility: match code commits with the corresponding data so you can
+  always go back to different versions of the full project (code, data, and
+  effects).
+- Snapshots: track and organize data and ML model versions with a simple CLI
+  (`dvc add` and `dvc checkout`, similar to basic Git commands) and `.dvc`
+  files.
+- Data as code: leverage Git workflow such as commits, branching, pull requests,
+  reviews, and even CI/CD for your data and models lifecycle. Think Git for
+  cloud storage, but without ad-hoc conventions.
+- Debugging: trace problems using the exact data that was used during
+  development.
+- Releasing: Tag stable data and models like you do with code. Employ semantic
+  versioning, connect branches to automatic deployment, etc.
+- Security: Git versioning of
+  [DVC metafiles](/doc/user-guide/dvc-files-and-directories) allows us to audit
+  data changes. And using cloud storage, data access control can be setup per
+  user or project.
+
+> ## ⚠️ DVC is not Git!
```

Comment:
we should do something about it- looks artificial. It's better for this to be clear from the introduction. One sentence should be enough I guess?

Reply:
Agreed.

Reply:
In fact it doesn't really belong in this use case any more. I moved it (shorter version) to the What is DVC? guide.